lib/partition_setup.pm

```diff
@@ -266,7 +266,9 @@ sub addlv {
 
     assert_screen 'expert-partitioner';
     send_key $cmd{system_view};
-    send_key 'home';
+    assert_screen_change(sub {
+            send_key 'home';
+    }, 5);
```

Comment:
I think it should be possible to rewrite it without `sub` and `;` in one line using `assert_screen_change { send_key 'home' },  5;` or perhaps with parenthesis for the function.

Reply:
Honestly, I do not know. testapi doc suggests using a callback when calling ``wait_screen_change`` with 2 arguments and since ``assert_screen_change`` is just wrapping ``wait_screen_change`` I have tried this approach. Is there any significant benefit to write it without callback ?

Reply:
Notice: If you use the second parameter, you could get the following warning

  Useless use of private variable in void context

To avoid it, use parentheses for the function call and the reserved word 'sub' for the callback
subroutine block.

Reply:
I didn't read that special case in the function without the wrapper :) thanks for the clarification, I was curious as I haven't tried that before, it is a pity it cannot be done in one line but there is no benefit more than avoid verbosity.